src/ffi.rs

```diff
@@ -29,6 +29,9 @@ use std::ptr;
 use std::slice;
 use std::sync::atomic;
 
+#[cfg(all(unix, feature = "qlog"))]
```

Comment:
Does it mean qlog doesn't work on Windows?

Reply:
this means specifically that the ability to pass in a raw FD for use with qlog is only available on unix platforms, which is a constraint of the function https://doc.rust-lang.org/std/os/unix/io/trait.FromRawFd.html